cmake/ActsCreatePackageConfig.cmake

```diff
@@ -26,6 +26,13 @@ install(
     DESTINATION ${install_package_config_dir}
 )
 
+# install third party FindXXX.cmake files
+file(GLOB_RECURSE _pckg_find_files "${CMAKE_CURRENT_LIST_DIR}/Find*.cmake")
+install(
+    FILES ${_pckg_find_files}
+    DESTINATION ${install_package_config_dir}/Modules
+)
```

Comment:
Wondering if it'd be better to list them explicitly, even if we just use a list for `_pckg_find_files`

Reply:
Hmm I see the advantage of finding them automatically is to not forget this as we in fact always do

Reply:
I don't think that is harmfull because
- For `find_package(xyz CONFIG)` this does not apply
- For `find_package(xyz)` this should resolve to the same as during build time, which is desired I think?